src/rules/completed-docs/tagExclusion.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * @license
+ * Copyright 2013 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ts from "typescript";
+
+import { Exclusion } from "./exclusion";
+
+export interface ITagExclusionDescriptor {
+    tags: {
+        content: {
+            [i: string]: string;
+        };
+        existence: string[];
+    };
+}
+
+export class TagExclusion extends Exclusion<ITagExclusionDescriptor> {
+    private readonly contentTags = this.descriptor.tags ? this.descriptor.tags.content : {};
+    private readonly existenceTags = new Set(this.descriptor.tags && this.descriptor.tags.existence);
+
+    public excludes(node: ts.Declaration) {
+        const documentationNode = this.getDocumentationNode(node);
+        const tagsWithContents = this.parseTagsWithContents(documentationNode.getFullText());
```

Comment:
@JoshuaKGoldberg `SyntaxWalker#visitNode` will never visit any JSDoc nodes, that's why I pointed you to `node.getChildren()` which does some magic to include jsdoc as the first entries if available.
However, I don't consider this a blocker for the PR. I'm fine with merging this without that change and tweaking it afterwards.